bin/ovpn_getclient

```diff
@@ -51,3 +75,29 @@ fi
 if [ -n "$OVPN_MTU" ]; then
     echo "tun-mtu $OVPN_MTU"
 fi
+}
+
+dir="$OPENVPN/clients/$cn"
+case "$parm" in
+    "separated")
+        mkdir -p "$dir"
+        get_client_config "$parm" > "$dir/${cn}.ovpn"
+        cp "$EASYRSA_PKI/private/${cn}.key" "$dir/${cn}.key"
+        cp "$EASYRSA_PKI/ca.crt" "$dir/ca.crt"
+        cp "$EASYRSA_PKI/issued/${cn}.crt" "$dir/${cn}.crt"
+        cp "$EASYRSA_PKI/dh.pem" "$dir/dh.pem"
+        cp "$EASYRSA_PKI/ta.key" "$dir/ta.key"
+        ;;
+    "combined")
+        get_client_config "combined"
+        ;;
+    "combined-save")
+        get_client_config "combined" > "$dir/${cn}-combined.ovpn"
+        ;;
+    *)
```

Comment:
Default case should be *combined* as mentioned above to not break existing tutorials.